docs/AdvancedFeatures.md

```diff
@@ -24,6 +24,7 @@ Function Signature | Description
 `startsWith(String str, String prefix)`| [`StringUtils.startsWith(...)`](https://commons.apache.org/proper/commons-lang/javadocs/api-3.3/org/apache/commons/lang3/StringUtils.html#startsWith(java.lang.CharSequence,%20java.lang.CharSequence))
 `length(String string)`| [`StringUtils.length(...)`](https://commons.apache.org/proper/commons-lang/javadocs/api-3.3/org/apache/commons/lang3/StringUtils.html#length(java.lang.CharSequence))
 `defaultIfEmpty(String str, String default)` | [`StringUtils.defaultIfEmpty(...)`](https://commons.apache.org/proper/commons-lang/javadocs/api-3.3/org/apache/commons/lang3/StringUtils.html#defaultIfEmpty(T,%20T))
+`escapeXml11(String str)` | [`StringEscapeUtils.escapeXml11(...)`](https://commons.apache.org/proper/commons-lang/javadocs/api-3.3/org/apache/commons/lang3/StringEscapeUtils.html#escapeXml11(java.lang.String))
```

Comment:
Sorry to only look at this now - @bpeptena I would prefer to expose this method as `escapeXml()` to the yaml files and I would keep the fact, that `StringEscapeUtils.escapeXml11()` is called under the hood an implementation detail (it's fine to mention it in the docu, but I would not force all users to write the method with `...11` in the yaml files. 

Especially because XML is not exactly a fast moving technology, I don't expect a `StringEscapeUtils.escapeXml12()` or even `StringEscapeUtils.escapeXml20()` to pop up soon ;-)

Reply:
There is a difference between XML 1.0 and 1.1 with respect to escape characters. We need the 1.0 variant for FileVault XML. Also making it explicit in the method name exposed in EL is IMHO a good thing.

Reply:
How should ACTool know the context? In some contexts XML 1.1 might make sense. The EL Parser doesn't really consider where this is being used (if with `initialContent` or somewhere else and we should IMHO leave it context free.